StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/AddPaymentMethodViewController.swift

```diff
@@ -315,6 +315,11 @@ class AddPaymentMethodViewController: UIViewController {
                 self.delegate?.updateErrorLabel(for: genericError)
             }
         }
+        
+        // TODO(porter) Revisit when there is a solution to ACHv2 in deferred mode
+        guard let intent = intent as? Intent else {
```

Comment:
Even though I added error checking earlier in PaymentSheet loading these are convenient to resolve compile time errors for now.